Neos.Flow/Classes/Session/Aspect/LazyLoadingAspect.php

```diff
@@ -34,9 +36,9 @@ class LazyLoadingAspect
 
     /**
      * @Flow\Inject
-     * @var SessionInterface
+     * @var SessionManagerInterface
      */
-    protected $session;
+    protected $sessionManager;
```

Comment:
What's the reason to inject `SessionManagerInterface` instead of `SessionInterface` here? Just for consistency with the components or is there a technical reason?

Reply:
Technical reason. This just like the AuthenticationProviderManager both are initialised very early in Flow (before the component chain runs), so the currentSession isn't initialised properly at this point, leading to cookies not being set.